### PR TITLE
readme: Update language support roadmap

### PR DESCRIPTION
**Pull request by kakkoyun**
We already have support for Ruby and Python.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,20 +170,19 @@ Flags:
 
 ## Roadmap
 
-* Additional language support for just-in-time (JIT) compilers, and dynamic languages (non-exhaustive list):
-  * Ruby
-  * Node.js
-  * Python
+* Additional language support for just-in-time (JIT) compilers and dynamic languages (non-exhaustive list):
   * JVM
```
Comment (javierhonduco) [inline README.md:174]: I believe we support the JIT bits with frame pointers

File: README.md [hunk 2]
```diff
@@ -170,20 +170,19 @@ Flags:
...
+  * Lua
+  * Node.js
```
Comment (javierhonduco) [inline README.md:176]: We technically support it too, no?

Comment (brancz) [reply README.md:176]: I would say we err on the side of "if it's possible at all we say we support it". So in that sense we do support JVM and nodejs. Not completely 100% zero-instrumentation yet, but with a tiny bit of effort people can already get it to work.

File: README.md [hunk 3]
```diff
@@ -170,20 +170,28 @@ Flags:
...
+* Additional **zero-instrumentation** language support for just-in-time (JIT) compilers and dynamic languages (non-exhaustive list):
+  * [JVM](https://www.parca.dev/docs/java-support)
+  * Lua
   * Node.js
-  * Python
-  * JVM
+
+
+  > [!IMPORTANT]
+  > All JIT compiled languages with frame pointer, and [Perf map](https://github.com/torvalds/linux/blob/master/tools/perf/Documentation/jit-interface.txt) or [jitdump](https://github.com/torvalds/linux/blob/master/tools/perf/Documentation/jitdump-specification.txt)
```
Comment (javierhonduco) [inline README.md:180]: This sentence feels incomplete, but might be me!

Comment (javierhonduco) [reply README.md:180]: Also, I feel this is too "technical" if this makes sense, we should point users to more user-friendly documentation. Most people enabling the flags for JIT symbolization probably don't know about the format

**Pull request merged**